### ISSUE TITLE
Mapping "user" to "author" directly

### ISSUE THREAD

**Issue by BigRoy** [mindbender-studio/config#4]
### Problem

In the current code during integration the `user` is mapped as `author` in the resulting data, this might introduce a slight confusion. As during publishing we're relying on _user_ being present as data whereas after it is _author_.

See: https://github.com/mindbender-studio/config/blob/e23348790c94570fb6a064b43a46f674a027e929/polly/plugins/publish/integrate_asset.py#L240

### Discussion

Should we maybe already collect this information as `author` from the beginning? Why, or why not?

**Comment by mottosso** [comment]
Makes sense.

The idea was for a single user to be able to "login" as someone else and publish from there, along with getting that user's working environment including tools and things. It's something I had at Framestore, though I didn't use it very much.

In that case, `user` to me is whomever is physically logged in to the OS, whereas `author` is the "virtually" logged in user.

**Comment by BigRoy** [comment]
> In that case, user to me is whomever is physically logged in to the OS, whereas author is the "virtually" logged in user.

Sure, to me the only odd thing is _for now_ that the naming of it changes in the integrator. If we don't do that we have less terminology at the moment to think about.

**Comment by mottosso** [comment]
Sure, let's patch that up. Let's go with "author"?

**Comment by BigRoy** [comment]
Yup, that should also make it easier to stay backwards compatible at this point as that was how it currently exists and persists in the database/storage.
